devnation-live/_common/devnationlive.yml

```diff
@@ -0,0 +1,120 @@
+    speaker: Sebastien Blanc
+    speaker_intro:
+        ""
+  - title: Blue/Green Deployments with OpenShift
+    date: 'Date TBD'
+    youtube_id:
+    abstract:
+        'The “deploy moment” is an occasion that still gives many developers the shivers.. Luckily enough, today we have tools like Kubernetes and OpenShift that enable us to turn the deploy moment into a simple, every day, every hour activity. Today we have Kubernetes and OpenShift to automate out-of-box many different scenarios like a Blue/Green deployment. Come and join this live demo session to learn how you can have a better and safe deployment using Linux containers.'
+    speaker: Rafael Benevides
+    speaker_intro:
+        ""
+  - title: Feature Toggles & Hypothesis Driven Development
+    date: 'Date TBD'
+    youtube_id:
+    abstract:
+        "Can you foresee the feature? Do you know if you’ll achieve your goals with the code you’re deploying today? Let’s drop this crystal ball and start practicing some Hypothesis-Driven Development. Join us on this webinar and we’ll show you how Feature Toggles can be bundled inside your software development process to enable you to quickly assess the business results of your code, and help you to make the decisions that matter."
+    speaker: Edson Yanaga
+    twitter_handle: 'yanaga'
+    speaker_intro:
+        ""
+  - title: Continuous Deployment at Scale
+    date: 'Date TBD'
+    youtube_id:
+    abstract:
+        "How do you manage CI/CD across hundreds of projects simultaneously? In this session, we will delve into a real-world example of a cloud-hosted SaaS based on hundreds of independent microservices, leveraging CI/CD and DevOps for a continuous flow to production."
+    speaker: James Rawlings
+    speaker_intro:
+        ""
+  - title: DevOps and ChatOps with OpenShift.io
+    date: 'Date TBD'
+    youtube_id:
+    abstract:
+        'The buzzword “DevOps” begins with Dev as in Developers.  In this session, we will demonstrate how a development team can quickly and easily setup a new Continuous Deployment Pipeline leveraging the best of open source microservices frameworks like Spring Boot, Vert.x and Wildfly Swarm as well as tools like Jenkins and Eclipse Che.'
+    speaker: James Strachan
+    speaker_intro:
+        ""
+  - title: Domain Driven Design for Mere Mortals
+    date: 'Date TBD'
+    youtube_id:
+    abstract:
+        ""
+    speaker: Justin Holmes
+    speaker_intro:
+        ""
+  - title: The IoT Meets Kubernetes
+    date: 'Date TBD'
+    youtube_id:
+    abstract:
+        "The the sheer volume of Internet of Things requires massively scalable ingestion and analytics engines and in this session you will see how to leverage messaging-as-a-service, the open source EnMasse project on a backplane of Cloud Native Computing Fundation’s Kubernetes."
+    speaker: Paolo Patierno
+    speaker_intro:
+        ""
```

Comment:
A number of these videos do not have the youtube Id. Will they be updated once they're known?